spacy/pipeline/spancat.py

```diff
@@ -185,25 +308,46 @@ def __init__(
         suggester: Suggester,
         name: str = "spancat",
         *,
+        add_negative_label: bool = False,
         spans_key: str = "spans",
-        threshold: float = 0.5,
+        negative_weight: Optional[float] = 1.0,
+        allow_overlap: Optional[bool] = True,
         max_positive: Optional[int] = None,
+        threshold: Optional[float] = 0.5,
         scorer: Optional[Callable] = spancat_score,
     ) -> None:
-        """Initialize the span categorizer.
+        """Initialize the multi-label or multi-class span categorizer.
+        The 'single_label' argument configures whether the component
```

Comment:
There are a number of out-of-date references to `single_label` in the docstring.

Reply:
ugh right, sorry, let me fix it!